apps/web/src/components/PreJoinScreens/PreJoinScreens.tsx

```diff
@@ -34,7 +34,7 @@ export default function PreJoinScreens() {
       if (appState.hasSpeakerInvite) {
         const { data } = await joinStreamAsSpeaker(appState.participantName, appState.eventName);
         await videoConnect(data.token);
-        chatConnect(data.token);
+        data.chat_enabled && chatConnect(data.token);
```

Comment:
Nit - maybe this can be an if statement like the others. Feels funny to use `&&`, even though it works the same.

Reply:
Thanks! Not sure why I handled this differently 🤔